api/lib/application/authentication/oidc/index.js

```diff
@@ -92,6 +94,38 @@ exports.register = async (server) => {
         tags: ['api', 'oidc', 'authentication'],
       },
     },
+    {
+      method: 'POST',
+      path: '/api/oidc/token-reconciliation',
+      config: {
+        auth: false,
+        pre: [
+          {
+            method: featureToggles.checkIfSsoAccountReconciliationIsEnabled,
+            assign: 'checkIfSsoAccountReconciliationIsEnabled',
+          },
+        ],
+        validate: {
+          payload: Joi.object({
+            data: Joi.object({
+              attributes: Joi.object({
+                email: Joi.string().email().required(),
+                password: Joi.string().required(),
+                'identity-provider': Joi.string().required().valid(POLE_EMPLOI, CNAV),
```

Comment:
```suggestion
                'identity-provider': Joi.string().required().valid('POLE_EMPLOI', 'CNAV'),
```

j'utliserai les valeurs de la nouvelles constantes apres, au lieu de rajouter qq ch que je vais juste enlever par la suite